### PR TITLE
chore: update staking doc

### PR DESCRIPTION
**Pull request by sainoe**
## Description

Add missing docs in staking REAMDE, see  [comment](https://github.com/cosmos/cosmos-sdk/pull/18694#discussion_r1435419676).

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

* [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] added `!` to the type prefix if API or client breaking change
* [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/main/CONTRIBUTING.md#pr-targeting))
* [ ] provided a link to the relevant issue or specification
* [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/build/building-modules/01-intro.md)
* [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/main/CONTRIBUTING.md#testing)
* [ ] added a changelog entry to `CHANGELOG.md`
* [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
* [ ] updated the relevant documentation or specification
* [ ] reviewed "Files changed" and left comments if necessary
* [ ] run `make lint` and `make test`
* [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

* [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] confirmed `!` in the type prefix if API or client breaking change
* [ ] confirmed all author checklist items have been addressed 
* [ ] reviewed state machine logic
* [ ] reviewed API design and naming
* [ ] reviewed documentation is accurate
* [ ] reviewed tests and test coverage
* [ ] manually tested (if applicable)

### CODE REVIEW

File: docs/architecture/adr-061-liquid-staking.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
-# ADR ADR-061: Liquid Staking
+# ADR 061: Liquid Staking Module
```
Comment (sainoe) [inline docs/architecture/adr-061-liquid-staking.md:1]: Updated ADR version from [SDK v0.45.16-ics-lsm](https://github.com/cosmos/cosmos-sdk/blob/v0.45.16-ics-lsm/docs/architecture/adr-061-liquid-staking.md).

File: x/staking/README.md [hunk 2]
```diff
@@ -328,6 +341,22 @@ they are in a deterministic order.
...
+
+* TotalLiquidStakedTokens: `0x85 -> math.Int`. 
+
+
+### PendingTokenizeShareAuthorizations
+
+PendingTokenizeShareAuthorizations stores a queue of addresses that have their tokenize share re-enablement/unlocking in progress. When an address is enqueued, it will sit for 1 unbonding period before the tokenize share lock is removed.
+
+```go reference
+https://github.com/cosmos/cosmos-sdk/blob/v0.45.16-ics-lsm/proto/cosmos/staking/v1beta1/staking.proto#L417-L421
```
Comment (mpoke) [inline x/staking/README.md:357]: Update link.

File: x/staking/README.md [hunk 3]
```diff
@@ -442,6 +474,8 @@ Redelegations affect the delegation, source and destination validators.
 * otherwise, if the `sourceValidator.Status` is not `Bonded`, and the `destinationValidator`
   is `Bonded`, transfer the newly delegated tokens from the `NotBondedPool` to the `BondedPool` `ModuleAccount`
 * record the token amount in an new entry in the relevant `Redelegation`
+* decrement the source validator's `LiquidShares`
```
Comment (mpoke) [inline x/staking/README.md]: Add the if condition, i.e., "if delegator is a liquid staking provider"

File: x/staking/README.md [hunk 4]
```diff
@@ -690,9 +738,15 @@ This message is expected to fail if:
...
 * the `Amount` `Coin` has a denomination different than one defined by `params.BondDenom`
+* the delegation is a `ValidatorBond` and the reduction in validator bond would cause the existing liquid delegation to exceed the cap.
+* the delegator is a liquid staking provider and the delegation exceeds
+either the `GlobalLiquidStakingCap`, the `ValidatorLiquidStakingCap` or the validator bond cap.
 
 When this message is processed the following actions occur:
 
+* if the delegation if is a validator bond, the `ValidatorBondShares` of the validator is decreased.
+* if the delegator is a liquid staking provider, the `TotalLiquidStakedTokens`` increased
+and the validator's `LiquidShares` is decreased.
```
Comment (mpoke) [inline x/staking/README.md]: Differentiate between the two validators.

File: x/staking/README.md [hunk 5]
```diff
@@ -705,6 +759,201 @@ When this message is processed the following actions occur:
 
 ![Begin redelegation sequence](https://raw.githubusercontent.com/cosmos/cosmos-sdk/release/v0.46.x/docs/uml/svg/begin_redelegation_sequence.svg)
 
+## MsgTokenizeShares
+
+The `MsgTokenizeShares` message allows users to tokenize their delegated tokens. Share tokens have denom using the validator address and record id of the underlying delegation with the format `{validatorAddress}/{recordId}`.
+
+```protobuf reference
+https://github.com/cosmos/cosmos-sdk/blob/v0.45.16-ics-lsm/proto/cosmos/staking/v1beta1/tx.proto#L49-L50
```
Comment (mpoke) [inline x/staking/README.md:767]: Update link.

File: x/staking/README.md (hunk 4 above)
Comment (sainoe) [reply x/staking/README.md]: Good catch.

**Review by mpoke — APPROVED** [review]

**Pull request merged**